modules/consentManagement.js

```diff
@@ -374,9 +519,20 @@ export function setConsentConfig(config) {
       utils.logError(`consentManagement config with cmpApi: 'static' did not specify consentData. No consents will be available to adapters.`);
     }
   }
-  if (!addedConsentHook) {
+
+  if (consentModule === 'usp' && !addedConsentHookUSP) {
+    $$PREBID_GLOBAL$$.requestBids.before(requestUspBidsHook, 50);
+    addedConsentHookUSP = true;
+  }
+
+  if (!addedConsentHook && consentModule === 'gdpr') {
     $$PREBID_GLOBAL$$.requestBids.before(requestBidsHook, 50);
+    addedConsentHook = true;
   }
```

Comment:
I don't think this handles the timeouts as is described in the spec (give each the maximum time possible), but a deal breaker.